webrender/src/device/gfx/device.rs

```diff
@@ -471,7 +471,12 @@ impl<B: hal::Backend> Device<B> {
             }
             .expect("create_command_pool_typed failed");
             unsafe { hal_cp.reset() };
-            let cp = CommandPool::new(hal_cp);
+            let mut cp = CommandPool::new(hal_cp);
+            cp.acquire_command_buffer();
+            /// Start recording for the 1st frame
```

Comment:
Use `//` comments instead of doc comments, because this generates warnings and fails the CI.

Reply:
Thank you! I've addressed the problem and rebased the PR to the master.